projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -177,17 +177,43 @@ function process_anchor_params() {
 	// Display an outbound link after publishing a post (only to English-speaking users since Anchor
 	// is English only).  This is only displayed if the blog is connected to an Anchor podcast.
 	$blog_connected_to_anchor = function_exists( 'get_blog_option' ) && false !== get_blog_option( get_current_blog_id(), 'anchor_podcast' );
+
+	/**
+	 * Allows disabling the Anchor convert to audio prompt.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param bool $is_enabled Whether the prompt is enabled or not.
+	 */
+	$is_enabled = apply_filters( 'jetpack_is_anchor_enabled', true );
+
 	if (
 		'post' === get_post_type() &&
 		! get_post_meta( $post->ID, 'jetpack_anchor_spotify_show', true ) &&
 		0 === strpos( get_user_locale(), 'en' ) &&
-		$blog_connected_to_anchor
+		$blog_connected_to_anchor &&
+		$is_enabled
 	) {
 		$data['actions'][] = 'show-post-publish-outbound-link';
 	}
 
 	wp_localize_script( 'jetpack-blocks-editor', 'Jetpack_AnchorFm', $data );
 }
 
+/**
+ * Function to check if the Anchor feature is enabled. This is used in the
+ * filter below.
+ *
+ * @return bool whether the feature is enabled or not.
+ */
+function is_anchor_enabled() {
+	return time() < strtotime( '2023-05-11 00:00:00 UTC' );
+}
```

Comment:
The filter should check whether the value has been overridden already, and not just return its own computation, otherwise overrides need to specify later priority to override this filter.
```suggestion
function is_anchor_enabled( $is_anchor_enabled ) {
	if ( ! $is_anchor_enabled ) {
		return false;
	}

	return time() < strtotime( '2023-05-11 00:00:00 UTC' );
}
```